golint scan failure

```
#docker run -ti --rm -v /disk1/sunyuan/tmp/terway/:/go/src/github.com/AliyunContainerService/terway -w /go/src/github.com/AliyunContainerService/terway pouchcontainer/pouchlinter:v0.1.2 bash -c "gometalinter --disable-all --skip vendor -E golint -d ./..."
DEBUG: [Jul  1 02:31:18.955] setenv PATH="/go/bin:/usr/local/go/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
DEBUG: [Jul  1 02:31:18.966] setenv GOROOT="/usr/local/go"
DEBUG: [Jul  1 02:31:18.966] Current environment:
DEBUG: [Jul  1 02:31:18.966] PATH="/go/bin:/usr/local/go/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
DEBUG: [Jul  1 02:31:18.966] GOPATH="/go"
DEBUG: [Jul  1 02:31:18.966] GOBIN=""
DEBUG: [Jul  1 02:31:18.966] GOROOT="/usr/local/go"
DEBUG: [Jul  1 02:31:18.967] linting path .
DEBUG: [Jul  1 02:31:18.967] linting path ./daemon
DEBUG: [Jul  1 02:31:18.967] linting path ./deviceplugin
DEBUG: [Jul  1 02:31:18.967] linting path ./examples/maxpods
DEBUG: [Jul  1 02:31:18.967] linting path ./pkg/aliyun
DEBUG: [Jul  1 02:31:18.967] linting path ./pkg/link
DEBUG: [Jul  1 02:31:18.967] linting path ./pkg/metric
DEBUG: [Jul  1 02:31:18.967] linting path ./pkg/pool
DEBUG: [Jul  1 02:31:18.967] linting path ./pkg/storage
DEBUG: [Jul  1 02:31:18.967] linting path ./pkg/tc
DEBUG: [Jul  1 02:31:18.967] linting path ./plugin/driver
DEBUG: [Jul  1 02:31:18.967] linting path ./plugin/terway
DEBUG: [Jul  1 02:31:18.967] linting path ./rpc
DEBUG: [Jul  1 02:31:18.967] linting path ./types
DEBUG: [Jul  1 02:31:18.967] linting path ./version
DEBUG: [Jul  1 02:31:18.973] [golint.1]: executing /go/bin/golint -min_confidence 0.800000 . ./daemon ./deviceplugin ./examples/maxpods ./pkg/aliyun ./pkg/link ./pkg/metric ./pkg/pool ./pkg/storage ./pkg/tc ./plugin/driver ./plugin/terway ./rpc ./types ./version
DEBUG: [Jul  1 02:31:19.335] [golint.1]: golint hits 6: ^(?P<path>.*?\.go):(?P<line>\d+):(?P<col>\d+):\s*(?P<message>.*)$
DEBUG: [Jul  1 02:31:19.335] nolint: parsing daemon/eni-multi-ip.go for directives
DEBUG: [Jul  1 02:31:19.335] [golint.1]: golint linter took 362.016088ms
DEBUG: [Jul  1 02:31:19.337] nolint: parsing daemon/eni-multi-ip.go took 1.957198ms
DEBUG: [Jul  1 02:31:19.337] nolint: parsing daemon/server.go for directives
daemon/eni-multi-ip.go:15:2:warning: const maxIpBacklog should be maxIPBacklog (golint)
daemon/eni-multi-ip.go:141:9:warning: if block ends with a return statement, so drop this else and outdent its block (golint)
DEBUG: [Jul  1 02:31:19.338] nolint: parsing daemon/server.go took 689.868µs
DEBUG: [Jul  1 02:31:19.338] nolint: parsing plugin/terway/cni.go for directives
daemon/server.go:14:2:warning: a blank import should be only in a main or test package, or have a comment justifying it (golint)
DEBUG: [Jul  1 02:31:19.340] nolint: parsing plugin/terway/cni.go took 2.141706ms
plugin/terway/cni.go:75:2:warning: don't use ALL_CAPS in Go names; use CamelCase (golint)
plugin/terway/cni.go:76:2:warning: don't use ALL_CAPS in Go names; use CamelCase (golint)
plugin/terway/cni.go:77:2:warning: don't use ALL_CAPS in Go names; use CamelCase (golint)
DEBUG: [Jul  1 02:31:19.340] total elapsed time 374.274954ms
```

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -33,7 +33,7 @@ jobs:
       - run:
           name: validate go code with gometalinter
           command: |
-            gometalinter --disable-all --skip vendor -E goconst -E gofmt -E ineffassign -E golint -E misspell -E vet -d ./...
+            gometalinter --disable-all --skip vendor -E goconst -E gofmt -E ineffassign -E goimports -E golint -E misspell -E vet -d ./...
       - run:
           name: use opensource tool client9/misspell to correct commonly misspelled English words 
           command: |

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -33,7 +33,7 @@ jobs:
       - run:
           name: validate go code with gometalinter
           command: |
-            gometalinter --disable-all --skip vendor -E goconst -E gofmt -E ineffassign -E misspell -E vet -d ./...
+            gometalinter --disable-all --skip vendor -E goconst -E gofmt -E ineffassign -E golint -E misspell -E vet -d ./...
       - run:
           name: use opensource tool client9/misspell to correct commonly misspelled English words 
           command: |

--- a/daemon/daemon.go
+++ b/daemon/daemon.go
@@ -3,6 +3,12 @@ package daemon
 import (
 	"encoding/json"
 	"fmt"
+	"io/ioutil"
+	"net"
+	"os"
+	"sync"
+	"time"
+
 	"github.com/AliyunContainerService/terway/pkg/aliyun"
 	"github.com/AliyunContainerService/terway/pkg/metric"
 	"github.com/AliyunContainerService/terway/pkg/pool"
@@ -12,13 +18,8 @@ import (
 	"github.com/pkg/errors"
 	log "github.com/sirupsen/logrus"
 	"golang.org/x/net/context"
-	"io/ioutil"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
-	"net"
-	"os"
-	"sync"
-	"time"
 )
 
 const (

--- a/daemon/eni-multi-ip.go
+++ b/daemon/eni-multi-ip.go
@@ -12,7 +12,7 @@ import (
 
 const (
 	maxEniOperating = 5
-	maxIpBacklog    = 10
+	maxIPBacklog    = 10
 )
 
 type eniIPFactory struct {
@@ -102,7 +102,7 @@ func (f *eniIPFactory) submit() error {
 			return nil
 		}
 	}
-	return errors.Errorf("trigger ENIIP throttle, max operating concurrent: %v", maxIpBacklog)
+	return errors.Errorf("trigger ENIIP throttle, max operating concurrent: %v", maxIPBacklog)
 }
 
 func (f *eniIPFactory) popResult() (ip *types.ENIIP, err error) {
@@ -138,9 +138,8 @@ func (f *eniIPFactory) Create() (ip types.NetworkResource, err error) {
 	if err == nil {
 		ip, err = f.popResult()
 		return
-	} else {
-		logrus.Debugf("allocate from exist eni error: %v, creating eni", err)
 	}
+	logrus.Debugf("allocate from exist eni error: %v, creating eni", err)
 
 	select {
 	case f.eniOperChan <- struct{}{}:
@@ -161,7 +160,7 @@ func (f *eniIPFactory) Create() (ip types.NetworkResource, err error) {
 	eni := &ENI{
 		ENI:       eniObj,
 		ecs:       f.eniFactory.ecs,
-		ipBacklog: make(chan struct{}, maxIpBacklog),
+		ipBacklog: make(chan struct{}, maxIPBacklog),
 		done:      make(chan struct{}, 1),
 	}
 
@@ -271,7 +270,7 @@ func newENIIPResourceManager(poolConfig *types.PoolConfig, ecs aliyun.ECS, alloc
 		eniFactory:   eniFactory,
 		enis:         []*ENI{},
 		eniOperChan:  make(chan struct{}, maxEniOperating),
-		ipResultChan: make(chan *ENIIP, maxIpBacklog),
+		ipResultChan: make(chan *ENIIP, maxIPBacklog),
 	}
 
 	capacity, err := ecs.GetInstanceMaxPrivateIP(poolConfig.InstanceID)
@@ -310,7 +309,7 @@ func newENIIPResourceManager(poolConfig *types.PoolConfig, ecs aliyun.ECS, alloc
 					ENI:       eni,
 					ips:       []*ENIIP{},
 					ecs:       ecs,
-					ipBacklog: make(chan struct{}, maxIpBacklog),
+					ipBacklog: make(chan struct{}, maxIPBacklog),
 					done:      make(chan struct{}, 1),
 				}
 				factory.enis = append(factory.enis, poolENI)

--- a/daemon/eni-multi-ip.go
+++ b/daemon/eni-multi-ip.go
@@ -2,12 +2,13 @@ package daemon
 
 import (
 	"fmt"
+	"sync"
+
 	"github.com/AliyunContainerService/terway/pkg/aliyun"
 	"github.com/AliyunContainerService/terway/pkg/pool"
 	"github.com/AliyunContainerService/terway/types"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
-	"sync"
 )
 
 const (

--- a/daemon/k8s.go
+++ b/daemon/k8s.go
@@ -3,6 +3,13 @@ package daemon
 import (
 	"encoding/json"
 	"fmt"
+	"net"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+	"unicode"
+
 	"github.com/AliyunContainerService/terway/deviceplugin"
 	"github.com/AliyunContainerService/terway/pkg/storage"
 	"github.com/pkg/errors"
@@ -12,12 +19,6 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/fields"
 	"k8s.io/client-go/kubernetes"
-	"net"
-	"os"
-	"strconv"
-	"strings"
-	"time"
-	"unicode"
 )
 
 const (

--- a/daemon/server.go
+++ b/daemon/server.go
@@ -2,15 +2,16 @@ package daemon
 
 import (
 	"fmt"
+	"io/ioutil"
+	"net"
+	"net/http"
+
 	"github.com/AliyunContainerService/terway/pkg/metric"
 	"github.com/AliyunContainerService/terway/rpc"
 	"github.com/pkg/errors"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 	log "github.com/sirupsen/logrus"
 	"google.golang.org/grpc"
-	"io/ioutil"
-	"net"
-	"net/http"
 
 	//import pprof for diagnose
 	_ "net/http/pprof"

--- a/daemon/server.go
+++ b/daemon/server.go
@@ -11,6 +11,8 @@ import (
 	"io/ioutil"
 	"net"
 	"net/http"
+
+	//import pprof for diagnose
 	_ "net/http/pprof"
 	"os"
 	"os/signal"

--- a/daemon/veth.go
+++ b/daemon/veth.go
@@ -3,6 +3,13 @@ package daemon
 import (
 	"context"
 	"fmt"
+	"io/ioutil"
+	"net"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
 	"github.com/AliyunContainerService/terway/pkg/link"
 	"github.com/AliyunContainerService/terway/types"
 	"github.com/containernetworking/plugins/plugins/ipam/host-local/backend/disk"
@@ -10,12 +17,6 @@ import (
 	"github.com/docker/docker/api/types/filters"
 	"github.com/docker/docker/client"
 	log "github.com/sirupsen/logrus"
-	"io/ioutil"
-	"net"
-	"os"
-	"path/filepath"
-	"strings"
-	"time"
 )
 
 const (

--- a/deviceplugin/eni.go
+++ b/deviceplugin/eni.go
@@ -1,21 +1,23 @@
 package deviceplugin
 
 import (
-	"k8s.io/apimachinery/pkg/util/wait"
 	"net"
 	"os"
 	"time"
 
+	"k8s.io/apimachinery/pkg/util/wait"
+
 	"fmt"
-	log "github.com/sirupsen/logrus"
-	"golang.org/x/net/context"
-	"google.golang.org/grpc"
 	"io/ioutil"
-	pluginapi "k8s.io/kubernetes/pkg/kubelet/apis/deviceplugin/v1beta1"
 	"path"
 	"regexp"
 	"sync"
 	"syscall"
+
+	log "github.com/sirupsen/logrus"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+	pluginapi "k8s.io/kubernetes/pkg/kubelet/apis/deviceplugin/v1beta1"
 )
 
 const (

--- a/examples/maxpods/maxpods.go
+++ b/examples/maxpods/maxpods.go
@@ -3,10 +3,11 @@ package main
 import (
 	"flag"
 	"fmt"
-	"github.com/AliyunContainerService/terway/pkg/aliyun"
-	"github.com/denverdino/aliyungo/common"
 	"io/ioutil"
 	"log"
+
+	"github.com/AliyunContainerService/terway/pkg/aliyun"
+	"github.com/denverdino/aliyungo/common"
 )
 
 var (

--- a/main.go
+++ b/main.go
@@ -2,6 +2,7 @@ package main
 
 import (
 	"flag"
+
 	"github.com/AliyunContainerService/terway/daemon"
 	log "github.com/sirupsen/logrus"
 )

--- a/pkg/aliyun/ecs.go
+++ b/pkg/aliyun/ecs.go
@@ -2,6 +2,10 @@ package aliyun
 
 import (
 	"fmt"
+	"net"
+	"sync"
+	"time"
+
 	"github.com/AliyunContainerService/terway/pkg/metric"
 	"github.com/AliyunContainerService/terway/types"
 	"github.com/denverdino/aliyungo/common"
@@ -9,9 +13,6 @@ import (
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 	"k8s.io/apimachinery/pkg/util/wait"
-	"net"
-	"sync"
-	"time"
 )
 
 // ECS the interface of ecs operation set

--- a/pkg/aliyun/eni.go
+++ b/pkg/aliyun/eni.go
@@ -3,13 +3,14 @@ package aliyun
 import (
 	"encoding/json"
 	"fmt"
+	"net"
+
 	"github.com/AliyunContainerService/terway/pkg/link"
 	"github.com/AliyunContainerService/terway/types"
 	"github.com/denverdino/aliyungo/common"
 	"github.com/denverdino/aliyungo/ecs"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
-	"net"
 )
 
 // ENIInfoGetter interface to get eni information

--- a/pkg/aliyun/metadata.go
+++ b/pkg/aliyun/metadata.go
@@ -2,12 +2,13 @@ package aliyun
 
 import (
 	"fmt"
-	"github.com/AliyunContainerService/terway/pkg/metric"
-	"github.com/denverdino/aliyungo/common"
 	"io/ioutil"
 	"net/http"
 	"strings"
 	"time"
+
+	"github.com/AliyunContainerService/terway/pkg/metric"
+	"github.com/denverdino/aliyungo/common"
 )
 
 const (

--- a/pkg/metric/util.go
+++ b/pkg/metric/util.go
@@ -1,8 +1,9 @@
 package metric
 
 import (
-	"github.com/prometheus/client_golang/prometheus"
 	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
 )
 
 // MsSince returns milliseconds since start.

--- a/pkg/pool/pool.go
+++ b/pkg/pool/pool.go
@@ -4,11 +4,12 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"github.com/AliyunContainerService/terway/types"
-	log "github.com/sirupsen/logrus"
 	"strings"
 	"sync"
 	"time"
+
+	"github.com/AliyunContainerService/terway/types"
+	log "github.com/sirupsen/logrus"
 )
 
 // Errors of pool

--- a/pkg/pool/pool_test.go
+++ b/pkg/pool/pool_test.go
@@ -3,11 +3,12 @@ package pool
 import (
 	"context"
 	"fmt"
-	"github.com/AliyunContainerService/terway/types"
-	"github.com/stretchr/testify/assert"
 	"sync"
 	"testing"
 	"time"
+
+	"github.com/AliyunContainerService/terway/types"
+	"github.com/stretchr/testify/assert"
 )
 
 type mockObjectFactory struct {

--- a/pkg/pool/queue_test.go
+++ b/pkg/pool/queue_test.go
@@ -2,10 +2,11 @@ package pool
 
 import (
 	"fmt"
-	"github.com/AliyunContainerService/terway/types"
-	"github.com/stretchr/testify/assert"
 	"testing"
 	"time"
+
+	"github.com/AliyunContainerService/terway/types"
+	"github.com/stretchr/testify/assert"
 )
 
 func TestInit(t *testing.T) {

--- a/pkg/storage/store.go
+++ b/pkg/storage/store.go
@@ -2,9 +2,10 @@ package storage
 
 import (
 	"errors"
+	"sync"
+
 	"github.com/boltdb/bolt"
 	log "github.com/sirupsen/logrus"
-	"sync"
 )
 
 // ErrNotFound key not found in store

--- a/pkg/tc/tc.go
+++ b/pkg/tc/tc.go
@@ -2,10 +2,11 @@ package tc
 
 import (
 	"fmt"
+	"math"
+
 	"github.com/pkg/errors"
 	log "github.com/sirupsen/logrus"
 	"github.com/vishvananda/netlink"
-	"math"
 )
 
 // TrafficShapingRule the interface traffic shaping rule

--- a/plugin/driver/drivers.go
+++ b/plugin/driver/drivers.go
@@ -2,14 +2,15 @@ package driver
 
 import (
 	"fmt"
+	"net"
+	"os"
+	"syscall"
+
 	"github.com/AliyunContainerService/terway/pkg/tc"
 	"github.com/containernetworking/cni/pkg/types"
 	"github.com/containernetworking/plugins/pkg/ns"
 	"github.com/pkg/errors"
 	"github.com/vishvananda/netlink"
-	"net"
-	"os"
-	"syscall"
 )
 
 // drivers implement objects

--- a/plugin/driver/ipvlan.go
+++ b/plugin/driver/ipvlan.go
@@ -2,11 +2,12 @@ package driver
 
 import (
 	"fmt"
+	"net"
+
 	"github.com/containernetworking/cni/pkg/types"
 	"github.com/containernetworking/plugins/pkg/ns"
 	"github.com/pkg/errors"
 	"github.com/vishvananda/netlink"
-	"net"
 )
 
 type ipvlanDriver struct {

--- a/plugin/driver/raw_nic.go
+++ b/plugin/driver/raw_nic.go
@@ -2,13 +2,14 @@ package driver
 
 import (
 	"encoding/hex"
+	"math/rand"
+	"net"
+	"time"
+
 	"github.com/containernetworking/cni/pkg/types"
 	"github.com/containernetworking/plugins/pkg/ns"
 	"github.com/pkg/errors"
 	"github.com/vishvananda/netlink"
-	"math/rand"
-	"net"
-	"time"
 )
 
 type rawNicDriver struct {

--- a/plugin/driver/utils.go
+++ b/plugin/driver/utils.go
@@ -2,10 +2,11 @@ package driver
 
 import (
 	"fmt"
+	"net"
+
 	"github.com/containernetworking/plugins/pkg/utils/sysctl"
 	"github.com/pkg/errors"
 	"github.com/vishvananda/netlink"
-	"net"
 )
 
 func deleteRoutesForAddr(addr *net.IPNet, tableID int) error {

--- a/plugin/terway/cni.go
+++ b/plugin/terway/cni.go
@@ -4,6 +4,10 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"net"
+	"runtime"
+	"time"
+
 	"github.com/AliyunContainerService/terway/pkg/link"
 	"github.com/AliyunContainerService/terway/plugin/driver"
 	"github.com/AliyunContainerService/terway/rpc"
@@ -17,9 +21,6 @@ import (
 	"github.com/pkg/errors"
 	"github.com/vishvananda/netlink"
 	"google.golang.org/grpc"
-	"net"
-	"runtime"
-	"time"
 )
 
 const (

--- a/rpc/rpc.pb.go
+++ b/rpc/rpc.pb.go
@@ -5,10 +5,11 @@ package rpc
 
 import (
 	fmt "fmt"
+	math "math"
+
 	proto "github.com/golang/protobuf/proto"
 	context "golang.org/x/net/context"
 	grpc "google.golang.org/grpc"
-	math "math"
 )
 
 // Reference imports to suppress errors if they are not otherwise used.

